README.md

```diff
@@ -120,13 +120,13 @@ You'll want to **unit test** the code if you're testing a new version or contrib
 We're on a 8/4 cycle which means 8 weeks of features and 4 weeks of stabilization
 capped at a release once every 3 months ie. at the last of the third month.
 
-Update `CORE_VERSION` in `CMakeLists.txt` to `8.0`.
-Add a section to `ChangeLog`.
+Update `CORE_VERSION` in `CMakeLists.txt` to `10.0`.
+Add a section to `CHANGELOG.md`.
 
-    git commit -p -v -m "Release Midori 8.0"
-    git checkout -B release-8.0
+    git commit -p -v -m "Release Midori 10.0"
```

Comment:
Why 10 instead of 9?

Reply:
So the commands can be used for the next release. It was wrong before.